vaadin-renderer-flow-parent/vaadin-renderer-flow/src/test/java/com/vaadin/flow/data/renderer/FormattedRenderersSerializableTest.java

```diff
@@ -0,0 +1,34 @@
+package com.vaadin.flow.data.renderer;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectOutputStream;
+
+import org.junit.Test;
+
+public class FormattedRenderersSerializableTest {
+
+    @Test
+    public void numberRendererIsSerializable() throws IOException {
+        final NumberRenderer renderer = new NumberRenderer(
+                value -> value.toString(), "");
+        new ObjectOutputStream(new ByteArrayOutputStream())
+                .writeObject(renderer);
+    }
```

Comment:
Not sure if this test is testing the fix correctly. If I rollback the changes on `NumberRenderer`, the test result is still green.

Reply:
Yes ... as NumberFormat is actually serializable and NumberRenderer is not needing the fix

public abstract class Format implements Serializable, Cloneable {
   ...
}

Reply:
I reverted changes on NumberRenderer, but I updated the unit test and propose to keep it for future sake.